### PR TITLE
Add a CLA check

### PR DESCRIPTION
**Pull request by dturn**
**What are you trying to accomplish with this PR?**
Comply with Shopify's OSS rules.

**How is this accomplished?**
Require CLA agreement via probots

**What could go wrong?**
More friction for contributors

### CODE REVIEW

**Comment by dturn** [comment]
Link to cla is https://cla.shopify.com/

This will appear as a new required validation:

Screen Shot 2019-06-19 at 9 24 55 AM

> Should we add something to our contributors' guide?

Yep, that's a good idea.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -105,6 +106,11 @@ This gem uses subclasses of `KubernetesResource` to implement custom success/fai
 6. Add a basic example of the type to the hello-cloud [fixture set](https://github.com/Shopify/kubernetes-deploy/tree/master/test/fixtures/hello-cloud) and appropriate assertions to `#assert_all_up` in [`hello_cloud.rb`](https://github.com/Shopify/kubernetes-deploy/blob/master/test/helpers/fixture_sets/hello_cloud.rb). This will get you coverage in several existing tests, such as `test_full_hello_cloud_set_deploy_succeeds`.
 7. Add tests for any edge cases you foresee.
 
+### Contributor License Agreement
+
+ New contributors will be required to sign [Shopify's Contributor License Agreement (CLA)](https://cla.shopify.com/).
+ There are two versions of the CLA one for individuals and one for organizations.
```
Comment (KnVerey) [inline CONTRIBUTING.md]:
```suggestion
 There are two versions of the CLA: one for individuals and one for organizations.
```
